src/utils/requestUtils.ts

```diff
@@ -78,8 +85,16 @@ export async function downloadProject(state: ProjectGenState): Promise<ZipFile>
 }
 
 async function tryGetProjectBuffer(projectUrl: string): Promise<Buffer> {
+  const requestOptions: request.OptionsWithUri = {
+    uri: projectUrl,
+    headers: {
+      'Client-Name': CLIENT_NAME,
+      'Client-Contact-Email': CLIENT_CONTACT_EMAIL
```

Comment:
is it possible to define a headers constant holding those values? to avoid repeating it for each query

Reply:
simply do 
```ts
const HEADERS = {
  'Client-Name': 'vscode-quarkus',
  'Client-Contact-Email': 'fbricon@redhat.com,azerr@redhat.com,dakwon@redhat.com'
};
...

const requestOptions: request.OptionsWithUri = {
    uri: `${apiUrl}/extensions`,
    headers: HEADERS,
    timeout: 30000
  };
...
```